### DISCUSSION TITLE
Service unavailablility due to maintenance

### DISCUSSION THREAD

**Discussion by oruscalleda** [INSPIRE-MIF/helpdesk-registry#63] [Announcements]
Dear Community:

Due to required maintenance on our buildings, the INSPIRE Registry service will be unavailable from **FRIDAY 11** August 12:00 until **WEDNESDAY 16** August 12:00.

Sorry for the inconvenience.

**Comment by jenriquesoriano** [comment]
Dear Community,

the INSPIRE Registry service is back to operation according to the planned maintenance.

Best regards,

**Comment by juanpelegrina** [comment]
Service is back in operation according to the planned maintenance.
